pygui/__init__.py

```diff
@@ -9,24 +9,22 @@
 from PySide6 import QtCore
 
 try:
+    # If rich is available, use it to improve traceback logs
     from rich.traceback import install
     import shutil
 
     terminal_width = shutil.get_terminal_size((100, 20)).columns - 2
-
     install(width=terminal_width)
```

Comment:
I would suggest keeping this out of the library and in your testing/example setup. You never know how this library will be used; users may end up having rich in their environment as a transient dependency, but still have other preferences for their output handling :)

Reply:
Good call, moved it!